velox/exec/LocalPlanner.cpp

```diff
@@ -83,6 +83,12 @@ OperatorSupplier makeConsumerSupplier(
       };
       return std::make_unique<CallbackSink>(operatorId, ctx, consumer);
     };
+  } else if (
+      auto join =
+          std::dynamic_pointer_cast<const core::HashJoinNode>(planNode)) {
```

Comment:
What is this change for? It is surprising to see a join logic in a diff that focuses on aggregation.